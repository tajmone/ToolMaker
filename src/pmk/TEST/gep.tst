set os = "$argv[1]"
if ("$os" == "PC") then
  dos2unix -ascii gep.tst.win32
else
  ../pmk gep
endif
