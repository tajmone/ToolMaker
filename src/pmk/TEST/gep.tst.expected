########## gep ##########
set os = "$argv[1]"
if ("$os" == "PC") then
  dos2unix -ascii gep.tst.win32
else
  ../pmk gep
endif
########## /bin/csh -f gep.tst UNIX ##########


gep.pmk

   25.  q = x ( y ! z ) ;
=====>            1

  *1*   102 E : '!' deleted.


1 error(s).
No detected warnings.

